201-decrypt-running-linux-vm/azuredeploy.json

```diff
@@ -0,0 +1,75 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "vmName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the virtual machine"
+      }
+    },
+    "volumeType": {
+      "type": "string",
```

Comment:
We absolutely want the customers to see this single value. This is for a number of reasons. We do not support decryption on OS drive for Linux VMs and once they load the template we want that to be explicit.